packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -222,7 +222,7 @@ export default class SdNavigationItem extends SolidElement {
               ? this.separated
                 ? html`<button
                     type="button"
-                    title="chevron-down-button"
+                    title="toggle-accordion"
```

Comment:
```suggestion
                    title="toggle-details"
```

Reply:
I am not 100% aware of the usecase for this title, but as we used summary and details down in the code, it kind of makes it strange to talk about accordions here?!?

Or am I missing something?

Reply:
we could also rename it with the actual purpose. As this is nothing generic but the navigation-item, we could also call it "toggle-navigation-details"

Reply:
Second that